From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeremy Apthorp <jeremya@chromium.org>
Date: Wed, 8 Jan 2020 16:06:14 -0800
Subject: FIXME: grit conflicts

This works around a bug in grit's handling of generated .grd files.

Should be removed once grit is fixed.

diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index 1d7e991d68b040b41101f20322eecfaf8a170e95..32af87cfb473a419f6ee5e54431e68ffbaa95514 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -566,15 +566,16 @@
     "includes": [3840],
   },
 
+  # FIXME: put this back after devtools_resources once grit is fixed to respect META properly
+  "electron/electron_resources.grd": {
+    "messages": [3860],
+    "includes": [4100],
+  },
+
   # This file is generated during the build.
   "<(SHARED_INTERMEDIATE_DIR)/devtools/devtools_resources.grd": {
     "META": {"sizes": {"includes": [500],}},
-    "includes": [3860],
-  },
-
-  "electron/electron_resources.grd": {
-    "messages": [31750],
-    "includes": [31950],
+    "includes": [4860],
   },
 
   # END "everything else" section.
